pallets/elections/src/lib.rs

```diff
@@ -539,6 +538,8 @@ pub mod pallet {
                 }
             }
 
+            Self::emit_fresh_bans_event();
```

Comment:
if this works it should be put in separate pr (with versions, spec_version etc bumped)

Reply:
It is out of this PR, then.